lib/chef/resource/remote_directory.rb

```diff
@@ -29,7 +29,7 @@ class RemoteDirectory < Chef::Resource::Directory
 
       provides :remote_directory
 
-      description "Use the **remote_directory** resource to incrementally transfer a directory from a cookbook to a node. The director that is copied from the cookbook should be located under COOKBOOK_NAME/files/default/REMOTE_DIRECTORY. The remote_directory resource will obey file specificity."
+      description "Use the **remote_directory** resource to incrementally transfer a directory from a cookbook to a node. The director that is copied from the cookbook should be located under `COOKBOOK_NAME/files/default/REMOTE_DIRECTORY`. The `remote_directory` resource will obey file specificity."
```

Comment:
```suggestion
      description "Use the **remote_directory** resource to incrementally transfer a directory from a cookbook to a node. The directory that is copied from the cookbook should be located under `COOKBOOK_NAME/files/default/REMOTE_DIRECTORY`. The `remote_directory` resource will obey file specificity."
```

Reply:
What does "incrementally" mean in this context?

Reply:
Great question. That's just copied word for word from the docs site.

Reply:
I didn't know what "file specificity" meant in this context, and when I looked it up I ended up on a Chef doc page. Maybe worth linking that? https://docs.chef.io/templates/#file-specificity